tests/WE2E/machine_suites/coverage.cheyenne

```diff
@@ -1,9 +1,9 @@
-custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_FALSE
+custom_ESGgrid_IndianOcean_6km
 grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot
 grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16
 grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR
 grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
 grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_HRRR_suite_HRRR
-#nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_GFS_v16
+nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_GFS_v16
```

Comment:
I think including "nco" in these test names is still necessary, since they are explicitly run in NCO mode unless "community" is specified on the command line (or if it's a a community-mode coverage test). Now, whether or not these tests need to still exist is another question; I don't think any of them are unique, so we could eliminate them altogether so long as all those capabilities are tested as part of the Hera NCO coverage suite. I think it's a good idea to remove the tests, so if you agree then I'll go ahead and do that. @MichaelLueken should probably approve of that plan as well.

Reply:
I take back one minor point: there is one thing that is only tested in the "nco_" tests, and that is using pre-staged grid/orog/sfc_climo data. I can't remember, is that something that should be specific to NCO tasks, or can we roll that into another test?

Reply:
@mkavulich A couple of things:
1) I see 4 tests that start with "nco_" under `ufs-srweather-app/tests/WE2E/test_configs/grids_extrn_mdls_suites_nco`.  I started checking if there are counterparts of these under `grids_extrn_mdls_suites_community` to see if the "nco_" tests can be eliminated but decided to not finish because it's a pretty time consuming task (at least a couple of hours).  I made an issue at #896.  Feel free to comment there if I didn't capture the issue correctly.
2) Use of pre-staged grid/orog/sfc_climo data is not specific to NCO mode, but I'd say it is more important for NCO mode because (at least as of a couple of years ago) EMC (who always seem to run in NCO mode) did not want to have the `make_[grid|orog|sfc_climo]` tasks as part of their workflow, i.e. they always used pre-staged data.  That's why NCO mode always uses pre-staged files (or at least used to; not sure now).  But these are also important in community mode because once community-mode users get their grid/orog/sfc_climo files set up, they tend to want to skip those tasks in later experiments.